File: SDKs/googleapis.com/bigtableadmin/v2/typescript/src/sdk/models/shared/union.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { GcRule } from "./gcrule";



// Union
/** 
 * A GcRule which deletes cells matching any of the given rules.
**/
export class Union extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=rules", elemType: GcRule })
  rules?: GcRule[];
}

File: SDKs/googleapis.com/bigtableadmin/v1/python/src/sdk/models/shared/union.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from sdk import utils
from . import *


@dataclass_json
@dataclass
class Union:
    r"""Union
    A GcRule which deletes cells matching any of the given rules.
    """
    
    rules: Optional[List[GcRule]] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('rules') }})
    
